src/browser/modules/Stream/CypherFrame/TableView.jsx

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2002-2017 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Component } from 'preact'
+import { v4 } from 'uuid'
+import { StyledStatsBar, PaddedTableViewDiv, StyledBodyMessage } from '../styled'
+import Ellipsis from 'browser-components/Ellipsis'
+import {StyledTable, StyledBodyTr, StyledTh, StyledTd, StyledJsonPre} from 'browser-components/DataTables'
+import bolt from 'services/bolt/bolt'
+import { deepEquals, shallowEquals } from 'services/utils'
+import { getBodyAndStatusBarMessages, getRecordsToDisplayInTable } from './helpers'
+
+const toTable = (records) => records && records.length ? bolt.recordsToTableArray(records) : undefined
+
+const renderCell = (entry) => {
+  if (Array.isArray(entry)) {
+    const children = entry.map((item, index) => <span>{renderCell(item)}{index === entry.length - 1 ? null : ', '}</span>)
+    return <span>[{children}]</span>
+  } else if (typeof entry === 'object') {
+    return renderObject(entry)
+  } else {
+    return entry
```

Comment:
There's a separate card for this, but maybe `JSON.stringify` will do?
![oskar4j 2017-06-27 at 09 11 36](https://user-images.githubusercontent.com/570998/27575025-bd7eb9e4-5b18-11e7-9abd-df5a6aaa009f.png)